create_dmd_release/patches/dlang.org.patch

```diff
@@ -25,22 +25,23 @@ index b18204f..745fc99 100644
  ################################################################################
  
  # stable dub and dmd versions used to build dpl-docs
-@@ -118,6 +114,14 @@ DDOC_VARS_STABLE=\
+@@ -126,6 +122,15 @@ DDOC_VARS_STABLE_HTML=\
  	  DOCSRC="$(PWD)" \
  	  VERSION="$(abspath ${DMD_DIR}/VERSION)"
  
-+DDOC_VARS_RELEASE=\
++DDOC_VARS_RELEASE_HTML=\
 +	  DOC_OUTPUT_DIR="${DOC_OUTPUT_DIR}/phobos" \
 +	  STDDOC="$(addprefix $(PWD)/, $(STD_DDOC))" \
 +	  DMD="$(abspath $(DMD))" \
++	  DMD_DIR="$(abspath ${DMD_DIR})" \
```

Comment:
DMD_DIR is needed since we include dmd/osmodel.mak from the recursive make invocation in .generated/phobos.